CMakeLists.txt

```diff
@@ -246,8 +247,12 @@ if (LLAMA_CUBLAS)
         set(GGML_SOURCES_CUDA ggml-cuda.cu ggml-cuda.h)
 
         add_compile_definitions(GGML_USE_CUBLAS)
+        add_compile_definitions(GGML_CUDA_FORCE_DMMV=${LLAMA_CUDA_FORCE_DMMV})
```

Comment:
This causes cmake to build with `-DGGML_CUDA_FORCE_DMMV=OFF`. Since what the code checks is whether the macro is defined and not the value, this won't work.